pkg/spec/createconfig.go

```
@@ -387,6 +387,14 @@ func (c *CreateConfig) getContainerCreateOptions(runtime *libpod.Runtime, pod *l
 		options = append(options, libpod.WithStaticIP(ip))
 	}
 
+	if c.MacAddress != "" {
+		mac, _ := net.ParseMAC(c.MacAddress)
```

Ooops, I ignore errors (`_`) but I test `err` to `nil` on the very next line.